packages/itwin/ec3-widget/package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "@itwin/ec3-widget-react",
+  "version": "0.0.0",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the EC3 Carbon Calculation.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "iTwin.js",
+    "Carbon Calculation"
+  ],
+  "main": "lib/cjs/ec3-widget-react.js",
+  "directories": {
```

Comment:
Remove `directories` property from package.json. Can't find anything about the `directories.test` property in documentation (https://docs.npmjs.com/cli/v9/configuring-npm/package-json).